_common

```diff
@@ -116,7 +116,8 @@ handle_unknown() {
         group_mailto=$(echo "$group_description" | perl -n -wE'm/.*MAILTO: (\S+).*/ and say $1' | head -1)
         group_mailto=${group_mailto:-$notification_address}
         if [[ -n "$group_mailto" ]]; then
-            echo "$header"$'\n'"Reason: $reason"$'\n'"$excerpt" | mailx -s "Unknown issue to be reviewed (Group $group_id)" -r "openqa-label-known-issues <$from_email>" "$group_mailto"
+            group_description=$(echo "$group_data" | runjq -r '.[0].name' ) || true
```

Comment:
Does this conflict with [L115](https://github.com/os-autoinst/scripts/pull/163/files#diff-d43d416085698e2b7d04ae7e385fec724b027e46fb9870fb76ada26a8bac931fR115)